configuration/assigning-devices.md

```diff
@@ -58,15 +58,15 @@ list of available devices, which you can select to be assigned to that VM.
 Finding the right USB controller
 --------------------------------
 
-If you want to assign a certain [USB] device to a VM (by attaching the whole
-USB controller), you need to figure out which PCI device is the right
+If you want to assign a certain USB device to a VM by attaching the whole
```

Comment:
By dropping parens i just meant ().

the [] around USB was because it linked to https://www.qubes-os.org/doc/usb/